src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -378,6 +513,117 @@ public PreferencesViewModel(PreferenceSettings preferenceSettings, DynamoPythonS
 
             //By Default the Default Run Settings radio button will be in Manual
             RunSettingsIsChecked = true;
+
+            //By Default the warning state of the Visual Settings tab (Group Styles section) will be disabled
+            isWarningEnabled = false;
+
+            StyleItemsList = new ObservableCollection<StyleItem>();
+          
+            //When pressing the "Add Style" button some controls will be shown with some values by default so later they can be populated by the user
+            AddStyleControl = new StyleItem() { GroupName = "", HexColorString = "#" + GetRandomHexStringColor() };
+
+            //This piece of code will populate all the description text for the RadioButtons in the Geometry Scaling section.
+            optionsGeometryScal = new GeometryScalingOptions();
+            optionsGeometryScal.EnumProperty = GeometryScaleSize.Medium;
+            optionsGeometryScal.DescriptionScaleRange = new ObservableCollection<string>();
+            optionsGeometryScal.DescriptionScaleRange.Add(string.Format(Res.ChangeScaleFactorPromptDescriptionContent, scaleRanges[GeometryScaleSize.Small].Item2,
+                                                                                              scaleRanges[GeometryScaleSize.Small].Item3));
+            optionsGeometryScal.DescriptionScaleRange.Add(string.Format(Res.ChangeScaleFactorPromptDescriptionContent, scaleRanges[GeometryScaleSize.Medium].Item2,
+                                                                                              scaleRanges[GeometryScaleSize.Medium].Item3));
+            optionsGeometryScal.DescriptionScaleRange.Add(string.Format(Res.ChangeScaleFactorPromptDescriptionContent, scaleRanges[GeometryScaleSize.Large].Item2,
+                                                                                              scaleRanges[GeometryScaleSize.Large].Item3));
+            optionsGeometryScal.DescriptionScaleRange.Add(string.Format(Res.ChangeScaleFactorPromptDescriptionContent, scaleRanges[GeometryScaleSize.ExtraLarge].Item2,
+                                                                                              scaleRanges[GeometryScaleSize.ExtraLarge].Item3));
+        }
+
+        /// <summary>
+        /// This method will remove the current Style selected from the Styles list
+        /// </summary>
+        /// <param name="groupName"></param>
+        public void RemoveStyleEntry(string groupName)
```

Comment:
@RobertGlobant20 I guess what I was curious was more for these utility functions, seems they can be internal since only used in `DynamoCoreWpf`

Reply:
@QilongTang I modified the accesibility level for the next public methods to internal located in the PreferencesViewModel class:

- void RemoveStyleEntry
- bool ValidateExistingStyle
- void ResetAddStyleControl()
- string GetRandomHexStringColor()

Check commit: https://github.com/DynamoDS/Dynamo/pull/11636/commits/af9a3ca1c40d6688754c48178d43fe9775699cb1
Thanks